addons/captives/functions/fnc_doBlindfoldCaptive.sqf

```diff
@@ -0,0 +1,43 @@
+#include "script_component.hpp"
+/*
+ * Author: mrschick
+ * Puts a blindfold on a captive unit if the player has a blindfold in their inventory
+ *
+ * Arguments:
+ * 0: _unit-Player <OBJECT>
+ * 1: target <OBJECT>
```

Comment:
You're missing 3rd argument here, although it's passed by the action I don't see that you use it anywhere.

Reply:
Yeah, that's a remnant of the original function `fnc_doEscortCaptive` which uses it, thanks for pointing it out 👍🏻